src/utils/include/utils/print.hpp

```diff
@@ -1,42 +0,0 @@
-/*
- * Copyright (C) 2017-2019 The ESPResSo project
- *
- * This file is part of ESPResSo.
- *
- * ESPResSo is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * ESPResSo is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
-#ifndef CORE_UTILS_PRINT_HPP
-#define CORE_UTILS_PRINT_HPP
-
-#include <iostream>
-
-namespace Utils {
-
-/**
- * @brief Python style print function.
- */
-template <typename T> void print(T v) { std::cout << v << '\n'; }
-
-/**
- * @brief Python style print function.
- */
-template <typename Arg, typename... Args> void print(Arg v, Args... args) {
```

Comment:
Can I keep that one? I use that for debugging...

Reply:
I can restore it if you need it, but if the file is no longer part of the git history, you could also keep an untracked copy. It'll not be added if you use `git add -u src` and rebase operations will ignore it. I have quite a few such untracked files lying around in my clone and never had an issue with them. You could add a commit hook just to be sure it's not getting accidentally added.

Reply:
@fweik you marked it as resolved because you'll use the untracked copy?